### PR TITLE
Add funcs for getting user information

### PR DESCRIPTION
**Pull request by ChrisTheBaron**

### CODE REVIEW

File: show.go [hunk 1]
```diff
@@ -112,13 +113,13 @@ func (s *Session) GetSeasons(id int) ([]Season, error) {
 		seasons[k].FirstTime, err = time.Parse("02/01/2006 15:04", v.FirstTimeRaw)
 
 		if err != nil {
-			return nil, err
+			log.Print(err)
```
Comment (CaptainHayashi) [inline show.go]: Why is this now printing these errors instead of returning them?

Comment (ChrisTheBaron) [inline show.go]: Because I wasn't sure what else to do, and I wanted to see it in the logs when data wasn't being loaded

Comment (LordAro) [inline show.go]: The program should decide what to do with the error, not the library

File: user.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+	"fmt"
+	"time"
+)
+
+type Officership struct {
+	OfficerId   uint   `json:"officerid,string"`
+	OfficerName string `json:"officer_name"`
+	TeamId      uint   `json:"teamid,string"`
+	FromDateRaw string `json:"from_date,omitempty"`
+	FromDate    time.Time
```
Comment (LordAro) [inline user.go]: I'd still prefer to find a different solution to these "Raw" types, but that is not for this prq

**Comment by LordAro** [comment]
Good. Needs some unit tests ;)

File: user.go (hunk 2 above)
Comment (ChrisTheBaron) [inline user.go]: The long term solution is to refactor the API to return all datetimes in a format that go handles without all this fuss :smile:

File: show.go (hunk 1 above)
Comment (ChrisTheBaron) [inline show.go]: Okay, will revert

**Pull request merged**